packages/gatsby/cache-dir/static-entry.js

```diff
@@ -275,6 +298,8 @@ export default (pagePath, callback) => {
   scripts
     .slice(0)
     .reverse()
+    // remove legacy scripts from our preload as we preload our modern files instead
```

Comment:
I'm unsure if there is a workaround for this that we can have our cake and eat it too. A way to do this is document.write but I rather don't want to use it.

Reply:
Added the note!